build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -107,7 +107,7 @@
     </Target>
 
     <Target Name="SetupNuGetPackagesArchiveInputsOutputs"
-            DependsOnTargets="Prepare;SetupStage;SetupAzureBlobInformation">
+            DependsOnTargets="Prepare;SetupAzureBlobInformation">
```

Comment:
Make sure to run an archive generation to make verify this does not break anything. But looking at the code, this seems like left over.